server/src/test/java/io/crate/integrationtests/CopyIntegrationTest.java

```diff
@@ -820,4 +821,78 @@ public void test_copy_from_csv_file_with_empty_string_as_null_property() throws
             printedTable(response.rows()),
             is("1| foo\n2| NULL\n3| NULL\n"));
     }
+
+    @Test
+    public void testCopyFromFileSchemeWithFileLevelWildCards() throws Exception {
```

Comment:
Would it be possible to write unit tests against `getUrisWithGlob` directly instead?

Would be a lot less expensive than running the full integration tests.

Reply:
@mfussenegger Hello Jordi I have made some changes accordingly. Please have another look.

1) I found out that my fix has inconsistency where for file scheme it kept trailing slash while s3 does not. I think it still works but made a fix for that.
2) removed most of the test cases, integration testing for file scheme and mocked s3 test cases. They are then replace by the `getUrisWithGlob` test.
3) in order to test `getUrisWithGlob` relaxed some of the visibilities.

Reply:
and I should revert the changes of 'S3FileInputTest` instead of just removing my test cases.